src/libs/NumberFormatUtils.ts

```diff
@@ -1,7 +1,10 @@
 import type {ValueOf} from 'type-fest';
 import type CONST from '@src/CONST';
+import intlPolyfill from './IntlPolyfill';
 import memoize from './memoize';
 
+intlPolyfill();
```

Comment:
nab, but could this have gone to it's own `ios.ts` file? Would that help?

Reply:
Changed ✅